shared-helpers/src/types/backend-swagger.ts

```diff
@@ -6267,31 +6301,6 @@ export interface PublicLotteryTotal {
   multiselectQuestionId?: string
 }
 
-export enum ListingViews {
-  "fundamentals" = "fundamentals",
-  "base" = "base",
-  "full" = "full",
-  "details" = "details",
-  "csv" = "csv",
-}
-
-export enum ListingOrderByKeys {
-  "mostRecentlyUpdated" = "mostRecentlyUpdated",
-  "applicationDates" = "applicationDates",
-  "mostRecentlyClosed" = "mostRecentlyClosed",
-  "mostRecentlyPublished" = "mostRecentlyPublished",
-  "name" = "name",
-  "waitlistOpen" = "waitlistOpen",
-  "status" = "status",
-  "unitsAvailable" = "unitsAvailable",
-  "marketingType" = "marketingType",
-}
-
-export enum OrderByEnum {
```

Comment:
I'm guessing the `generate:client` command generated this change. However, when I ran it locally it converted it back to this format. And I believe this is style we want. I'm curious what would have caused this change for you locally. I vaguely this was a problem at one point for enums and I don't remember what changed. Potentially you have locally a different version of some dependency compared to my machine